nixos/modules/services/x11/desktop-managers/maxx.nix

```diff
@@ -13,6 +13,12 @@ in {
   config = mkIf (xcfg.enable && cfg.enable) {
     environment.systemPackages = [ pkgs.maxx ];
 
+    # there is hardcoded path in binaries
+    system.activationScripts.setup-maxx = ''
+      mkdir -p /opt
+      ln -sfn ${pkgs.maxx}/opt/MaXX /opt
```

Comment:
That is not very pretty. Is there no other way to solve this?

Reply:
@markuskowa Using `libredirect`, but that way minimized icons are totally broken. Also MaXX author strongly suggests using /opt/MaXX location.

Reply:
It's unfortunate that the package is closed source.